tushare/stock/trading.py

```diff
@@ -676,7 +676,8 @@ def get_k_data(code=None, start='', end='',
                            ignore_index=True)
     if ktype not in ct.K_MIN_LABELS:
         if ((start is not None) & (start != '')) & ((end is not None) & (end != '')):
-            data = data[(data.date >= start) & (data.date <= end)]
+            if data.empty==False:       

```

Comment:
They serve the same purpose.
Do we need to use English in forking this Chinese project? I am a newbie, so please kindly inform me. Thanks.